pull_request_template.md

```diff
@@ -38,3 +37,9 @@ If you have any questions or problems, please give a comment in the PR.
 - [ ] The choice of the type of test (unit test or systemtest) is reasonable
 - [ ] Testname matches exactly what is being tested
 - [ ] On a fail, output of the test leads quickly to the origin of the fault
+
+##### CHANGELOG.md
+- [ ] Separate commit for this PR in the CHANGELOG.md
+- [ ] The changelog entries refers to this PR (verify the number)
+- [ ] New changelog entries are added at the top of a section.
+- [ ] Commit message is "updated CHANGELOG.md" or "updated CHANGELOG.md for PR #0000" (replace 0000 by the number of the current PR)
```

Comment:
```suggestion
- [ ] Commit message is "update CHANGELOG.md" or "update CHANGELOG.md for PR #0000" (replace 0000 by the number of the current PR)
```

Reply:
refer to the developer guideline for git commits: use imperative language.

But as said above, I would not want to split the CHANGELOG.md checkmark into four submarks.